src/test/java/bio/terra/service/snapshotbuilder/SnapshotBuilderTestData.java

```diff
@@ -179,24 +184,97 @@ private static SnapshotBuilderProgramDataListOption generateSnapshotBuilderProgr
                       .name("All surveys")
                       .featureValueGroupName("Surveys")));
 
+  public static final Column PERSON_ID_COLUMN =
+      new Column().name("person_id").type(TableDataType.INTEGER);
+  public static final Column RACE_CONCEPT_ID_COLUMN =
+      new Column().name(Person.RACE_CONCEPT_ID).type(TableDataType.INTEGER);
+  public static final Column GENDER_CONCEPT_ID_COLUMN =
+      new Column().name(Person.GENDER_CONCEPT_ID).type(TableDataType.INTEGER);
+  public static final Column ETHNICITY_CONCEPT_ID_COLUMN =
+      new Column().name(Person.ETHNICITY_CONCEPT_ID).type(TableDataType.INTEGER);
+  public static final Column YEAR_OF_BIRTH_COLUMN =
+      new Column().name(Person.YEAR_OF_BIRTH).type(TableDataType.INTEGER);
+
+  public static final DatasetTable PERSON_TABLE =
+      new DatasetTable()
+          .name(Person.TABLE_NAME)
+          .columns(
+              List.of(
+                  PERSON_ID_COLUMN,
+                  RACE_CONCEPT_ID_COLUMN,
+                  GENDER_CONCEPT_ID_COLUMN,
+                  ETHNICITY_CONCEPT_ID_COLUMN,
+                  YEAR_OF_BIRTH_COLUMN));
+
+  public static final Column CONCEPT_ID_COLUMN =
+      new Column().name(Concept.CONCEPT_ID).type(TableDataType.INTEGER);
+  public static final Column CONCEPT_NAME_COLUMN =
+      new Column().name(Concept.CONCEPT_NAME).type(TableDataType.STRING);
+  public static final DatasetTable CONCEPT_TABLE =
+      new DatasetTable()
+          .name(Concept.TABLE_NAME)
+          .columns(List.of(CONCEPT_ID_COLUMN, CONCEPT_NAME_COLUMN));
+
+  private static final Column DRUG_CONCEPT_ID_COLUMN =
+      new Column().name(DrugExposure.DRUG_CONCEPT_ID).type(TableDataType.INTEGER);
+  private static final Column DRUG_TYPE_CONCEPT_ID_COLUMN =
+      new Column().name(DrugExposure.DRUG_TYPE_CONCEPT_ID).type(TableDataType.INTEGER);
+  private static final Column DRUG_ROUTE_CONCEPT_ID_COLUMN =
+      new Column().name(DrugExposure.DRUG_ROUTE_CONCEPT_ID).type(TableDataType.INTEGER);
+  private static final Column DRUG_SOURCE_CONCEPT_ID_COLUMN =
+      new Column().name(DrugExposure.DRUG_SOURCE_CONCEPT_ID).type(TableDataType.INTEGER);
+
+  public static final DatasetTable DRUG_TABLE =
+      new DatasetTable()
+          .name(DrugExposure.TABLE_NAME)
+          .columns(
+              List.of(
+                  PERSON_ID_COLUMN,
```

Comment:
The columns will be empty in the asset and access request. 
However, this chunk of code is populating the dataset itself. Since we have relationships between columns and a root column in asset definition, it would complain if the dataset itself didn't have any columns.